src/Indicator.vala

```diff
@@ -31,6 +31,8 @@ public class Session.Indicator : Wingpanel.Indicator {
     private Wingpanel.Widgets.Button log_out;
     private Wingpanel.Widgets.Button suspend;
     private Wingpanel.Widgets.Button shutdown;
+    private Wingpanel.Widgets.Button? settings = null;
```

Comment:
Don't declare those instance variables here, they aren't needed for one time initialization.